features/specific/predicate_message.feature

```diff
@@ -0,0 +1,70 @@
+@wip
+Feature: Predicate Message Feature
+
+  This is just a simple feature meant to test the fix for issue #81.
+  Problem was Cucumber-Cpp didn't recoginize failed asserations
+  when using following Boost macros:
```

Comment:
Rephrase without the issue please. I.e. something like "Support Boost.Test's different validation and reporting macros..."

Reply:
done